.env.example

```diff
@@ -60,6 +60,17 @@ AWS_VALUE=secret
 AWS_VALUE2=secret
 AWS_CLOUD_FORMATION=url
 
+# AWS credentials (Access Key and Secret Key): Configure AWS credentials in the AWS CLI profile using the 'aws configure' command.
+AWS_PROFILE=
+BUCKET_OWNER_ID=
```

Comment:
Rename `BUCKET_OWNER_ID` to `S3_BUCKET_OWNER_ID` so it is clear we are talking about S3 and not some other kind of buckets.  Also please add a comment to say what should go here (the AWS account ID of the bucket owner.)